### PR TITLE
Connector Health: Fixed SAT for GitHub, Google Analytics v4 (UA), Google Search Console, Hubspot, Recharge, Sendgrid, Greenhouse, Chargebee, Survey Monkey, Twitter, Zenloop, Zendesk-support, Pinterest, Slack, Sentry

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
Fixed SAT for GitHub, Google Analytics v4 (UA), Google Search Console, Hubspot, Recharge, Sendgrid, Greenhouse, Chargebee, Survey Monkey, Twitter, Zenloop, Zendesk-support, Pinterest, Slack, Sentry. 
## How
Updated expected records and SAT config.

### CODE REVIEW

**Review by davydov-d — APPROVED** [review]
Please bump version and update the changelog of the Instagram source since the python code has been changed

File: airbyte-integrations/connectors/source-instagram/source_instagram/streams.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def _state_has_legacy_format(self, state: Mapping[str, Any]) -> bool:
 
     def get_updated_state(self, current_stream_state: MutableMapping[str, Any], latest_record: Mapping[str, Any]):
         """Update stream state from latest record"""
-        record_value = latest_record[self.cursor_field]
+        record_value = latest_record.get(self.cursor_field, self._start_date)
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-instagram/source_instagram/streams.py:254]: As Denys suggested: please bump this connector version as you changed its code.

File: airbyte-integrations/connectors/source-recharge/integration_tests/expected_records.jsonl [hunk 2]
```diff
@@ -14,7 +14,7 @@
...
...essor": "shopify_payments", "platform_domain": "airbyte.myshopify.com", "shop_email": "integration-test@airbyte.io", "shop_phone": "1111111111", "subscriptions_enabled": 1, "test_mode": false, "timezone": "(GMT+02:00) Europe/Zaporozhye", "updated_at": "Wed, 08 Mar 2023 14:02:33 GMT"}, "store": {"checkout_logo_url": null, "checkout_platform": "shopify", "created_at": "Wed, 21 Apr 2021 11:44:38 GMT", "currency": "USD", "customer_portal_domain": "", "disabled_currencies_historical": [], "domain": "airbyte.myshopify.com", "email": "integration-test@airbyte.io", "enabled_presentment_currencies": ["USD"], "enabled_presentment_currencies_symbols": [{"currency": "USD", "location": "before", "suffix": "", "symbol": "$"}], "external_platform": "shopify", "iana_timezone": "Europe/Zaporozhye", "id": 126593, "my_shopify_domain": "airbyte.myshopify.com", "name": "airbyte", "payment_processor": "shopify_payments", "platform_domain": "airbyte.myshopify.com", "shop_email": "integration-test@airbyte.io", "shop_phone": "1111111111", "subscriptions_enabled": 1, "test_mode": false, "timezone": "(GMT+02:00) Europe/Zaporozhye", "updated_at": "Wed, 08 Mar 2023 14:02:33 GMT"}}, "emitted_at": 1675769381290}
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-recharge/integration_tests/expected_records.jsonl:17]: Please format this file to get a newline at the end.

File: airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml [hunk 3]
```diff
@@ -52,6 +52,8 @@ acceptance_tests:
               bypass_reason: "Data changing frequently, because we use live account."
             - name: position
               bypass_reason: "Data changing frequently, because we use live account."
+            - name: country
+              bypass_reason: "Data changing frequently, because we use live account."
```
Comment (alafanechere) [inline airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:56]: Do we plan to not use a live account? There are a lot of ignored fields that drastically lower the coverage of the tests. cc @YowanR

**Review by alafanechere — CHANGES_REQUESTED** [review]
Mainly requesting changes because I think that tests are not passing for  hubspot and zendesk.

**Review by bazarnov — APPROVED** [review]

**Review by erohmensing — CHANGES_REQUESTED** [review]
@lazebnyi Please remove the changes to instagram in this PR. As a result of [this on call issue](https://github.com/airbytehq/oncall/issues/1640) we realized we think they are transient and we need a different way to handle them. This change will mask the error without fixing the underlying problem. 

@davydov-d said the errors were no longer reproducible with the test data due to transience, hopefully they won't bother you anymore, but if they do, feel free to link to that issue

File: airbyte-integrations/connectors/source-instagram/source_instagram/streams.py (hunk 1 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-instagram/source_instagram/streams.py]: Fix to instagram deleted. Resolved.

File: airbyte-integrations/connectors/source-recharge/integration_tests/expected_records.jsonl (hunk 2 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-recharge/integration_tests/expected_records.jsonl:17]: Updated

File: airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml (hunk 3 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml:56]: I think in future we should do that. Already inform @YowanR about that.

**Review by erohmensing — APPROVED** [review]
My requested change (removing changes to instagram connector) are implemented, so removing my blocking review, but not endorsing other changes I haven't reviewed

**Review by alafanechere — APPROVED** [review]
Approving to unblock as my original concerns are now solved.

**Pull request merged**